pkgs/development/python-modules/robotframework/default.nix

```diff
@@ -1,4 +1,6 @@
-{ lib, fetchFromGitHub, buildPythonPackage, jsonschema }:
+{ lib, fetchFromGitHub, buildPythonPackage,
+  tkinter, xmlschema, docutils, pygments, pyyaml, enum34, enum-compat, pillow, lxml, jsonschema,
+  python }:
```

Comment:
'python' is unused, can be removed.

Reply:
I think tkinter is used for the Dialog library (or something), what are the other deps for? (Perhaps say something about that in the commit message.)